common/scripts/common_cross_toolchain.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+check_arch() {
+	if [[ ! ${ARCH+x} ]]; then
+		# ARCH are not set
+		if [[ ! ${CROSS_COMPILE+x} ]]; then
+			# ARCH and CROSS_COMPILE are not set, we set ARCH to arm64 by default
+			export ARCH=arm64
```

Comment:
Should the main build scripts be based on `ARCH=AARCH64|ARM` or based on `ARCH=aarch64|arm`? Some packages built here handle the former, linux expects the later.